### PR TITLE
add slider component

### PR DESCRIPTION
**Pull request by pankaj-prog**
slider

### CODE REVIEW

File: src/components/slider/slider.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            <a href="/src/components/typography/typography.html">Typography</a>
+          </li>
+          <li><a href="#">Colors</a></li>
+        </ul>
+        <ul class="hz-nav sec-nav">
+          <li class="site-nav-link">
+            <a href="/src/components/avatar/avatar.html"> Avatar</a>
+          </li>
+          <li class="site-nav-link">
+            <a href="/src/components/alert/alert.html"> Alert </a>
```
Comment (theVivek-kumar) [inline src/components/slider/slider.html:39]: this is good very clean code

**Review by theVivek-kumar — APPROVED** [review]

File: src/components/slider/slider.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+              step="20"
+              min="0"
+              max="100"
+              value="0"
+            />
+          </div>
+
+          <div class="slider-output">
+            Output : <span id="slider-output">0</span>
+          </div>
```
Comment (manojsarna) [inline src/components/slider/slider.html:110]: slider looks good!!

File: src/components/slider/slider.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                <span>0</span><span>20</span><span>40</span><span>60</span
+                ><span>80</span><span>100</span>
+              </div>
+              <input
+                id="slider"
+                class="slider"
+                type="range"
+                step="20"
+                min="0"
+                max="100"
```
Comment (manojsarna) [inline src/components/slider/slider.html:126]: try to customize slider element

File: src/components/slider/slider.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const slider = document.getElementById("slider");
+const sliderOutput = document.getElementById("slider-output");
+
+slider.addEventListener("change", (e) => {
+  console.log(slider);
+  sliderOutput.innerText = e.target.value;
+});
```
Comment (manojsarna) [inline src/components/slider/slider.js]: looks good!!

Comment (manojsarna) [reply src/components/slider/slider.js]: try to remove console.log from the js file , before deploying

File: src/components/slider/slider.css [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  justify-content: space-between;
+}
+
+.slider {
+  width: 98%;
+  cursor: pointer;
+}
+
+.slider:active {
+  cursor: grabbing;
```
Comment (manojsarna) [inline src/components/slider/slider.css]: try to use only one cursor type!!

**Review by manojsarna — APPROVED** [review]
Do Check the comments 
you can merge !!

File: src/components/slider/slider.js (hunk 4 above)
Comment (pankaj-prog) [reply src/components/slider/slider.js]: okay

File: src/components/slider/slider.css (hunk 5 above)
Comment (pankaj-prog) [reply src/components/slider/slider.css]: okay i will update it

File: src/components/slider/slider.html (hunk 2 above)
Comment (pankaj-prog) [reply src/components/slider/slider.html:110]: Thank you

**Pull request merged**